web_src/js/features/tag-name-editor.js

```diff
@@ -0,0 +1,30 @@
+import $ from 'jquery';
+import {hideElem, showElem} from '../utils/dom.js';
+
+export function initTagNameEditor() {
+  const el = document.getElementById('tag-name-editor');
+  if (!el) return;
+
+  const existingTags = JSON.parse(el.getAttribute('data-existing-tags'));
+  if (!Array.isArray(existingTags)) return;
+
+  const defaultTagHelperText = el.getAttribute('data-tag-helper');
+  const newTagHelperText = el.getAttribute('data-tag-helper-new');
+  const existingTagHelperText = el.getAttribute('data-tag-helper-existing');
+
+  $('#tag-name').on('keyup', (e) => {
```

Comment:
https://docs.gitea.io/en-us/guidelines-frontend/#framework-usage

It's better to avoid mixing jQuery and Vanilla JS together which it reduces the maintainability.

Reply:
Thanks for your reviewing. I've removed jQuery and only use Vanilla JS.